.github/file-filters.yml

```diff
@@ -0,0 +1,4 @@
+# This is used by the action https://github.com/dorny/paths-filter
+
+high_risk_code: &high_risk_code
+  - ".github/file-filters.yml"
```

Comment:
- this is for testing and will be removed before merging
- need to align on set of potentially higher-risk files before merging as well
- use full repo paths, can check [getsentry/sentry-cocoa@main/.github/file-filters.yml](https://github.com/getsentry/sentry-cocoa/blob/main/.github/file-filters.yml?rgh-link-date=2024-09-30T06%3A52%3A52Z) for inspiration